docs/data/enumerations/district_type.yaml

```diff
@@ -0,0 +1,46 @@
+name: DistrictType
+tags:
+- _name: city
+  description: A city.
+- _name: city-council
+  description: A specific seat/jurisdiction for a city, town, or village council.
+- _name: congressional
+  description: A United States congressional district.
+- _name: county
+  description: A county.
+- _name: county-council
+  description: A county council district, either in its entirety or for a specific
+    seat.
+- _name: judicial
+  description: A judicial district.
+- _name: municipality
+  description: A civil division which is not a town, city, village, or county.
+- _name: national
+  description: The United States.
+- _name: school
+  description: A school district.
+- _name: special
+  description: A `special-purpose district`_ that exist separate from general-purpose
```

Comment:
@jdmgoogle The processing of these strings is "dumb" in the sense that the strings are included in the resulting reST file as is.  So the links will work in the resulting reST provided that the hyperlink targets are also defined.

In this case, the reST looks like this in the generated table--

    | school         | A school district.                                 |
    +----------------+----------------------------------------------------+
    | special        | A `special-purpose district`_ that exist separate  |
    |                | from general-purpose districts.                    |
    +----------------+----------------------------------------------------+
    | state          | A state, district, commonwealth, or U.S.           |

And the table is included as follows in the reST:

    .. include:: ../../tables/enumerations/district_type.rst

    .. _`special-purpose district`: http://en.wikipedia.org/wiki/Special-purpose_district
    .. _town: http://en.wikipedia.org/wiki/Town#United_States
    .. _`Wikipedia article`: http://en.wikipedia.org/wiki/Town#United_States

As you can see, the needed hyperlink target appears after the inclusion of the table.

I also checked that Sphinx runs without errors when building the HTML, and also that the links work when browsing locally to that generated HTML.